file/permCheck.go

```diff
@@ -0,0 +1,31 @@
+package file
```

Comment:
On file names, shouldn't we use `_` or `-` separation for file names?

Reply:
using snake case for file names